sourcemash/frontend/static/scss/sourcemash.scss

```diff
@@ -327,48 +322,50 @@ a.button-collapse.top-nav {
     }
 }
 
-.card-action {
-  @extend .grey-text;
+.item-card {
```

Comment:
I noticed you didn't make any style changes on feed-cards. Everything looks good on that end (e.g. feed descriptions don't run too long and take over the card, subscribe buttons don't get pushed by long titles, etc)?